packages/web/src/app/contexts/api/moonpay/constants.ts

```diff
@@ -1,4 +1,4 @@
 export const MOONPAY_SIGNER_URL = process.env.REACT_APP_MOONPAY_SIGNER_URL;
-export const MOONPAY_API_KEY = process.env.REACT_APP_MOONPAY_API_KEY;
+export const MOONPAY_API_KEY = String(process.env.REACT_APP_MOONPAY_API_KEY);
```

Comment:
I think if we add `as string` here, we will not need to add non-null assertion everywhere else (`!`)